File: Android/src/com/rtsmsproject/www1/classes/Status.java
package com.rtsmsproject.www1.classes;

public class Status {
    // 1 == Safe Zone, 0 == Danger Zone
    private Boolean AlertType;
    private Double Distance;
    // 1 == Miles, 0 == KM
    private Boolean UnitTypeDistance;

    public Boolean getAlertType() {
        return AlertType;
    }
    public void setAlertType(Boolean asOf) {
    	AlertType = asOf;
    }
    public Double getDistance() {
        return Distance;
    }
    public void setDistance(Double asOf) {
    	Distance = asOf;
    }
    public Boolean getUnitTypeDistance() {
        return UnitTypeDistance;
    }
    public void setUnitTypeDistance(Boolean asOf) {
    	UnitTypeDistance = asOf;
    }

}

File: WebService/RTSMS-WebServiceVer1.2.4/Classes/Status.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Runtime.Serialization;

namespace RTSMSWebService.Classes
{
    [DataContract]
    public class Status
    {
        // 1 == Safe Zone, 0 == Danger Zone
        [DataMember]
        public Boolean AlertType { get; set; }
        [DataMember]
        public Double Distance { get; set; }
        [DataMember]
        // 1 == Miles, 0 == KM
        public Boolean UnitTypeDistance { get; set; }
    }
}